contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -299,6 +299,11 @@ def risk(self) -> list[dict[str, Any]]:
                 risk_object['threat_object_field'] = entity.name
                 risk_object['threat_object_type'] = "url"
                 risk_objects.append(risk_object)
+            
+            elif 'Attacker' in entity.role:
+                risk_object['threat_object_field'] = entity.name
+                risk_object['threat_object_field'] = entity.type.lower()
```

Comment:
Should this actually be `risk_object['threat_object_type'] = entity.type.lower()` ?

Reply:
Yup, thanks!